docs/services/general-info/env-var-deltas/4.0.0-5.0.0-added.md

```diff
@@ -177,8 +177,8 @@
-|                                               | `OCIS_EVENTS_AUTH_USERNAME;SETTINGS_EVENTS_AUTH_USERNAME`                                                                                                                                          | The username to authenticate with the cache. Only applies when store type 'nats-js-kv' is configured.                                                                                                                                                                                                                                                                                                                                                                       |                                       |
-|                                               | `OCIS_EVENTS_AUTH_PASSWORD;SETTINGS_EVENTS_AUTH_PASSWORD`                                                                                                                                          | The password to authenticate with the cache. Only applies when store type 'nats-js-kv' is configured.                                                                                                                                                                                                                                                                                                                                                                       |                                       |
+|                                               | `OCIS_CACHE_AUTH_USERNAME;SETTINGS_CACHE_AUTH_USERNAME`                                                                                                                                            | The username to authenticate with the cache. Only applies when store type 'nats-js-kv' is configured.                                                                                                                                                                                                                                                                                                                                                                       |                                       |
+|                                               | `OCIS_CACHE_AUTH_PASSWORD;SETTINGS_CACHE_AUTH_PASSWORD`                                                                                                                                            | The password to authenticate with the cache. Only applies when store type 'nats-js-kv' is configured.                                                                                                                                                                                                                                                                                                                                                                       |                                       |
```

Comment:
I think that these are the correct names - they exist in `services/settings/pkg/config/config.go`